pkgs/servers/minio/default.nix

```diff
@@ -18,7 +23,7 @@ buildGoModule rec {
   subPackages = [ "." ];
 
   patchPhase = ''
-    sed -i "s/Version.*/Version = \"${version}\"/g" cmd/build-constants.go
+    sed -i "s/Version.*/Version = \"${versionTimestamp}\"/g" cmd/build-constants.go
```

Comment:
```suggestion
    sed -i "s/Version.*/Version = \"${lib.replaceStrings [":"] ["-"] version}\"/g" cmd/build-constants.go
```

Reply:
That wouldn't change the version, wouldn't it? `"2021-04-22T15-44-28Z"` -> replace `: / -` -> still `"2021-04-22T15-44-28Z"` which would be rejected by the minio browser.